### ISSUE TITLE
Индикация выполнения автоматических операций

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#915]
Originally reported on Google Code with ID 915

```
Время выполнения автоматических операций может быть достаточно долгим. Но
при  их запуске никакой индикации выполнения не появляется. Надо, хотябы
чтобы указатель мышки менялся на песочные часы.
```

Reported by `danilchyk` on 2008-02-18 09:31:19

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2008-05-14 14:48:44
- **Labels added**: Type-Enhancement
- **Labels removed**: Type-Defect

**Comment by gsbelarus** [comment]
```
Можно наверное сделать вообще по красивому, сделать новый блок, в нём 
создавать форму которая будет отображать анимацию. Тогда при разработке, 
можно было бы самому настраивать данное дело.
```

Reported by `Alexander.GoldenSoft` on 2008-05-14 15:51:04

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2009-03-26 11:13:42
- **Labels added**: GedeminExe

**Comment by gsbelarus** [comment]
```

  Кстати, последний пост очень дельный.

```

Reported by `NikolayUkleyko` on 2013-08-23 15:10:06

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-12-19 07:17:33
- **Status changed**: `Duplicate`
- **Merged into**: #3013
